Expensify/App
# [No QA] Connect New Workspace Page with the backend

### Details
Removes stuff that won't ship in V1 and connects the form submission with the backend.

### Fixed Issues

Fixes https://github.com/Expensify/Expensify/issues/167378

### Tests
1. Open Expensify.cash
2. Go to the `/workspace/new` route
3. Introduce a name and press the "Get Started" button
4. Open the developer console, you should see this error:
![image](https://user-images.githubusercontent.com/6759612/122432961-0e7c4000-cf96-11eb-95f3-8f7f4a300716.png)

### QA Steps
❌ 

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/6759612/122434180-27d1bc00-cf97-11eb-9df7-cbedfc803493.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/6759612/122434216-30c28d80-cf97-11eb-94b7-e467110f1707.png)

#### Desktop
![image](https://user-images.githubusercontent.com/6759612/122435814-9bc09400-cf98-11eb-9fb4-2fd9ac4376ab.png)
#### iOS
![image](https://user-images.githubusercontent.com/6759612/122436094-d4f90400-cf98-11eb-9d3f-10211bf66bc7.png)
#### Android
![image](https://user-images.githubusercontent.com/6759612/122436551-34571400-cf99-11eb-91b0-0cf195f0d86e.png)
